pkg/sink/codec/avro/avro.go

```diff
@@ -1015,9 +1008,18 @@ func NewBatchEncoderBuilder(ctx context.Context,
 	changefeedID model.ChangeFeedID,
 	config *common.Config,
 ) (codec.RowEventEncoderBuilder, error) {
-	schemaM, err := NewAvroSchemaManager(ctx, config.AvroSchemaRegistry, nil)
-	if err != nil {
-		return nil, errors.Trace(err)
+	var schemaM SchemaManager
+	var err error
+	if config.AvroConfluentSchemaRegistry != "" {
```

Comment:
is  there still other schema registry?

Reply:
There are three main Schema Registry options :Confluent Schema Registry, Aws Glue, Apicurio.

Now TiCDC only support Confluent Schema Registry.

Reply:
This check here is to decide whether the decoder should use Confluent Schema Registry or Aws Glue.